Q9/pashmeen Q9 solution

```diff
@@ -0,0 +1,30 @@
+// "static void main" must be defined in a public class.
```

Comment:
@pashmeen27 , checkout README and CONTRIBUTION files and follow all the guidelines regarding submitting a PR